pkg/apis/config/v1alpha1/adminqos.go

```diff
@@ -186,6 +186,12 @@ type EvictionConfig struct {
 	// +optional
 	DryRun []string `json:"dryRun"`
 
+	// MetricInsurancePeriod defines the metric insurance period for eviction manager and plugins.
+	// If set this period, eviction plugin will ignore the metric data that timestamp is older than
+	// this period.
+	// +optional
+	MetricInsurancePeriod *metav1.Duration `json:"metricInsurancePeriod,omitempty"`
+
```

Comment:
the CRD yaml file should be changed too.

Reply:
done